README.md

```diff
@@ -14,88 +26,150 @@ To get started with the Ada Trader project follow these steps:
+  - See a list of open orders in the left side of the bottom panel, including the:
+    - Symbol for each order
+    - Buy/Sell indicator for each order
+    - [Target price](#trading-vocabulary) for each order
+    - Cancel button for each order
+  - See an order entry form in the right side of the bottom panel, including the:
+    - Symbol selection drop-down, which lists all symbols shown in the quote ticker
+    - Target price input box
+    - Buy button
+    - Sell button
+  - Click the Cancel button on each order in the open orders list
+  - Click the Buy button in the order entry form
+  - Click the Sell button in the order entry form
+
+As a user, when I:
+  - Click the Cancel button for an order:
+    - That order is removed from the open orders list
+    - That order will never execute
+  - Click Buy in the order entry form:
+    - If the target price is blank OR is **greater than** the current market price:
+      - That order is not created and (TODO: error display?)
+    - If the target price is **less than or equal** to the current market price:
+      - A new Buy order is added to the bottom of the open orders list, with the:
+        - Symbol from the the form
+        - Target price from the form
+  - Click Sell in the order entry form:
+    - If the target price is blank OR is **less than** the current market price:
+      - That order is not created and (TODO: error display?)
+    - If the target price is **greater than or equal to** the current market price:
+      - A new Sell order is added to the bottom of the open orders list, with the:
+        - Symbol from the the form
+        - Target price from the form
+
+Additionally, when:
+  - The price of a stock quote changes:
+    - If there are any open Sell orders in the open orders list, where the symbol matches the quote:
+      - Each order executes, in the same manner as though the user had clicked "Sell" on the quote
+      - Each order is removed from the open order list
+      - Each order will never execute again
+    - If there are any open Buy orders in the open orders list, where the symbol matches the quote:
+      - Each order executes, in the same manner as though the user had clicked "Buy" on the quote
+      - Each order is removed from the open order list
+      - Each order will never execute again
+
+### Testing
+You should have tests for any validations your models have, as well as any custom functions that you create on those models. **Optional**: Write a test which verifies that limit orders are executed and destroyed when the relevant stock reaches the order's target price.
+
+### Advice
+In order to get the list of symbols for the order entry form's drop-down, you may need to have that view access the quote collection.
+
+When removing a Backbone View from the page, that does not mean that the associated Model is gone! Be careful when cancelling an order, because an order that isn't shown on the page might still "hear" events that are triggered.
```

Comment:
They're going to need advice on how to spot that error, I feel it's likely a subtle error that won't be visible immediately.

Reply:
I think this is currently mentioned in the Backbone Views of Models & Events textbook resource, but we should definitely find a way to reiterate it or make it more prominent.